docs/sources/shared/example.md

```diff
@@ -24,7 +24,7 @@ Below is an example from the docs, with backslashes added. The initial spaces ar
 
 ```
 \{{< docs/list >}}
-  \{{< docs/shared "manage-users/view-server-user-list.md" >}}
```

Comment:
commented on one of the other PRs, not sure we want shortcode in this list...maybe we do.